package.json

```diff
@@ -40,24 +40,26 @@
   ],
   "devDependencies": {
     "@aws-sdk/types": "^3.0.0",
-    "@aws-sdk/util-hex-encoding": "^3.1.0",
-    "@aws-sdk/util-utf8-browser": "^3.1.0",
+    "@aws-sdk/util-hex-encoding": "^3.29.0",
+    "@aws-sdk/util-utf8-browser": "^3.29.0",
+    "@aws-sdk/util-buffer-from": "^3.29.0",
     "@types/chai": "^4.2.12",
     "@types/mocha": "^7.0.2",
     "@types/node": "^14.0.27",
     "@types/sinon": "^9.0.4",
     "chai": "^4.2.0",
+    "dot-prop": "^6.0.1",
```

Comment:
In #166 it says to bump dot-prop but here makes it seem like we are picking up a new dependency. Is this actually the case or are we picking it up

Reply:
I am not dead certain... the only way I can force the package-lock to resolve to dot-prop 6 is by having this here. Normally I would not add dependencies to the package.json without very good reason. Without this, dot-prop was resolving to something much lower... I think 4.2.1... which is what #166 is targeting, but it was actually lower than what we already have... so I locked it on greater than 6.